Dockerfile

```diff
@@ -28,7 +28,6 @@ RUN mkdir -p reports
 RUN mkdir -p tests
 
 # Workaround for permissions in CI if run with a different user
-RUN chmod 777 -R /home/seluser/
```

Comment:
If you are deleting this line, then you have the delete the comment above as well 💡